bindings/nodejs/src/lib.rs

```diff
@@ -122,6 +123,14 @@ impl ToNapiValue for Value {
             databend_driver::Value::Variant(s) => String::to_napi_value(env, s),
             databend_driver::Value::Geometry(s) => String::to_napi_value(env, s),
             databend_driver::Value::Geography(s) => String::to_napi_value(env, s),
+            databend_driver::Value::Interval(i) => {
```

Comment:
Add this type mapping to `README.md` for the two bindings?

Reply:
> Add this type mapping to `README.md` for the two bindings?

Now databend-query is planing to support this type(not finis). Better to waiting interval type finish.

This pr need to merge first, because sqllogictest need upgrade bendsql version to support interval type.